runtime/caml/stack.h

```diff
@@ -52,95 +44,41 @@
 #ifdef TARGET_s390x
 #define Saved_return_address(sp) *((intnat *)((sp) - SIZEOF_PTR))
 #define Trap_frame_size 16
-#define Callback_link(sp) ((struct caml_context *)((sp) + Trap_frame_size))
 #endif
 
 #ifdef TARGET_arm
 #define Saved_return_address(sp) *((intnat *)((sp) - 4))
-#define Callback_link(sp) ((struct caml_context *)((sp) + 8))
 #endif
 
 #ifdef TARGET_amd64
+/* Size of the gc_regs structure, in words.
+   See amd64.S and amd64/proc.ml for the indices */
+#define Wosize_gc_regs (13 /* int regs */ + 16 /* float regs */)
 #define Saved_return_address(sp) *((intnat *)((sp) - 8))
-#define Callback_link(sp) ((struct caml_context *)((sp) + 16))
 #endif
 
 #ifdef TARGET_arm64
 #define Saved_return_address(sp) *((intnat *)((sp) - 8))
-#define Callback_link(sp) ((struct caml_context *)((sp) + 16))
+#define Context_needs_padding /* keep stack 16-byte aligned */
```

Comment:
Unused currently. It was needed earlier for the arm64 backend. May need it again when arm64 is revived.

Reply:
Tracking this for post-merge fixing in https://github.com/ocaml/ocaml/issues/10861